src/broadpeak.js

```
@@ -201,6 +201,12 @@ class BroadPeak extends BasePlugin {
         this.dispatchEvent(this.player.Event.SOURCE_URL_SWITCHED, {originalUrl: playbackUrl, updatedUrl});
         this._srcPromise.resolve();
       } else {
+        if (result.getErrorCode() === 3500) {
```

there's no cases where we don't want to ignore this error ?